stubs/extend-repository-styelguide.stub

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Styleguide\Repositories;
+
+use App\Repositories\PromosExtendedRepository as Repository;
+use Faker\Factory;
+use Factories\GenericPromo;
+
+class PromosExtendedRepository extends Repository
+{
+    /**
+     * Construct the factory.
+     */
+    public function __construct(Factory $faker)
+    {
+        $this->faker = $faker->create();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getRequestData(array $data)
+    {
+        /*
+        // Extend a Styleguide repository
+        $promoRepoRequestData = app()->make(PromoRepository::class)->getRequestData($data);
```

Comment:
Can remove the `make()` and make it become
`app(PromoRepository::class)->getRequestData($data)`